views/reports.xml.ejs

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<% var query_string_this = helpers.q({page: parseInt(page), query: query, inspector: inspector}); -%>
+<% var query_string_first = helpers.q({page: 1, query: query, inspector: inspector}); -%>
+<% var query_string_next = helpers.q({page: parseInt(page) + 1, query: query, inspector: inspector}); -%>
+<% var query_string_prev = helpers.q({page: parseInt(page) - 1, query: query, inspector: inspector}); -%>
+<% var last_page = Math.max(1, Math.floor((results.hits.total + 9) / 10)); -%>
+<% var query_string_last = helpers.q({page: last_page, query: query, inspector: inspector}); -%>
+<feed xmlns="http://www.w3.org/2005/Atom">
+<title>Oversight.io | Bringing into sight the US government's oversight community.</title>
+<link href="https://oversight.io/reports?<%= query_string_this %>"/>
+<updated><%= new Date().toISOString() %></updated>
+<id>https://oversight.io/reports.xml</id>
+<link rel="alternate" type="text/html" hreflang="en" href="https://oversight.io/reports?<%= query_string_this %>"/>
+<link rel="self" type="application/atom+xml" href="https://oversight.io<%= self_url %>"/>
+<link rel="first" href="https://oversight.io/reports.xml?<%= query_string_first %>"/>
+<% if (page > 1) { -%>
+<link rel="previous" href="https://oversight.io/reports.xml?<%= query_string_prev %>"/>
+<% } -%>
+<% if (results.hits.total > page * 10) { -%>
+<link rel="next" href="https://oversight.io/reports.xml?<%= query_string_next %>"/>
+<% } -%>
+<link rel="last" href="https://oversight.io/reports.xml?<%= query_string_last %>"/>
+<% var hits = results.hits.hits; -%>
+<% for (var i = 0; i < hits.length; i++) { -%>
+<% var report = hits[i]._source; -%>
+<% var highlight = hits[i].highlight; -%>
+<% var link = encodeURI('https:/' + '/oversight.io/report/' + report.inspector + '/' + report.report_id); -%>
```

Comment:
I'm not entirely sure, but I think it was that http://... got interpreted as a line comment. Without that, I was getting bizarre compilation errors.

```
Unexpected token ILLEGAL in /home/pi/oversight.io/views/reports.xml.ejs while compiling ejs
```